radio/src/translations/CMakeLists.txt

```diff
@@ -3,12 +3,16 @@ foreach(translation ${translations})
   get_filename_component(language ${translation} NAME_WE)
   get_filename_component(path ${translation} DIRECTORY)
   set(target ${language}.h)
+  set(tlanguage ${language})
+  if(NOT(PCB STREQUAL HORUS) AND language STREQUAL "cz")
+    set(tlanguage cz_reduced)
```

Comment:
I just played it safe,  I wasn't sure changing language would not bring havoc somewhere else!